packages/edit-site/src/components/page-pages/index.js

```diff
@@ -19,19 +19,23 @@ import { useState, useEffect, useMemo } from '@wordpress/element';
 import Page from '../page';
 import Link from '../routes/link';
 import PageActions from '../page-actions';
-import { DataViews, PAGE_SIZE_VALUES } from '../dataviews';
+import { DataViews } from '../dataviews';
 
 const EMPTY_ARRAY = [];
 const EMPTY_OBJECT = {};
 
 export default function PagePages() {
-	const [ reset, setResetQuery ] = useState( ( v ) => ! v );
-	const [ globalFilter, setGlobalFilter ] = useState( '' );
-	const [ paginationInfo, setPaginationInfo ] = useState();
-	const [ { pageIndex, pageSize }, setPagination ] = useState( {
-		pageIndex: 0,
-		pageSize: PAGE_SIZE_VALUES[ 0 ],
+	const [ view, setView ] = useState( {
+		type: 'table',
```

Comment:
view `type` or `layout`. This is going to be an important naming choice.

Reply:
I think this should be `list`, and the `ListView` is a Block List view, but the ship has sailed now probably 😄 .

Reply:
Fine by me